templates/layouts/aw-install-simg.hbs

```diff
@@ -15,58 +15,85 @@
       <p>Before installing AsteroidOS to your watch, make sure you are aware of the capabilities and limitations of AsteroidOS on the {{title}}. The following table should summarize the current support of this watch:</p>
       {{> body }}
       <h1>Preparation</h1>
-      <div class="install-preparation-box">
-          <img src="{{assets}}/img/{{deviceName}}.jpg" class="install-preparation-img"><br>
-      </div>
       <div class="install-preparation-box">
         <h3>Download AsteroidOS 1.0</h3>
         <a class="btn btn-primary" href="https://release.asteroidos.org/1.0/{{deviceName}}/asteroid-image-{{deviceName}}.ext4" role="button">asteroid-image-{{deviceName}}.ext4</a>
         <a class="btn btn-primary" href="https://release.asteroidos.org/1.0/{{deviceName}}/zImage-dtb-{{deviceName}}.fastboot" role="button">zImage-dtb-{{deviceName}}.fastboot</a>
-        <h3>Install Fastboot & Adb</h3>
+        <br><br><i>Place the files in your users "Downloads" folder for the later commands to work</i><br>
+        <h3>Install Fastboot & ADB</h3>
+        <b>Linux systems</b> with apt package manager
+        <br><br>
         <pre><code class="bash">sudo apt install android-tools-adb android-tools-fastboot android-tools-fsutils</code></pre>
-        <p><i>Note: previous command may differ depending on your OS</i></p>
+        <br>
+        <b>Windows users</b> download this installer <a href="https://forum.xda-developers.com/showthread.php?t=2317790">published on XDA by shimp208</a>
+        <br><br>
+        <a class="btn btn-primary" href="https://androidfilehost.com/?fid=746010030569952951" role="button">Minimal ADB and Fastboot 1.43</a>
+        <br><br>
+        <i>Start the .exe file with Administrator rights (via right click).</i>
+        <br><br>To convert the ext2 image to simg format, download a ext2simg containing zip, <a href="https://forum.xda-developers.com/showthread.php?t=2600364">e.g. from this XDA thread by A.S._id</a>
+        <br><br>
+        <a class="btn btn-primary" href="https://forum.xda-developers.com/showpost.php?p=49235638&postcount=5" role="button">ANDROID_IMG_REPACK_TOOLS - CYGWIN</a>
+        <br><br>
+        <i>Extract the zip to the folder you installed "Minimal ADB and Fastboot".</i>
+        <br>
+      </div>
+      <div class="install-preparation-box">
```

Comment:
Having such a long text next to the install-prepartion-box makes it look *very* odd. We should definitely reorder the content or move that image somewhere else.